### DISCUSSION TITLE
New Request: Mobile App, Desktop App, IDE App

### DISCUSSION THREAD

**Discussion by NewMan2024** [danny-avila/LibreChat#3949] [Feature Requests & Suggestions]
Hello
Initializing a request for a cross-platform app or native app, for mobile and desktop and IDE. It is better to have flexible architecture to have alternatives such as mobile and desktop. what do you think?

**Comment by leikoilja** [comment]
I have just open-sourced a Mac desktop app wrapper called LibreChat-UI [here](https://github.com/leikoilja/LibreChat-UI) if you are interested. I've been using it for the past 3 weeks and it has been working great for me :)
